src/glang/codegen/builtin_types.py

```diff
@@ -81,15 +86,15 @@ def __init__(
         name: str,
         specialization: list[SpecializationItem],
         definition: TypeDefinition,
-        alias: Optional[Type],
+        alias: Type | None,
```

Comment:
Hmm, I'm not convinced... Maybe I'll warm to it

Reply:
lol, we can change that back if you want

Reply:
It fine. You've made the change now. Is it caught by the linter or was this a manual search?

Reply:
It's caught by the linter